fv3core/stencils/del2cubed.py

```diff
@@ -41,34 +42,37 @@ def corner_fill(q_in: FloatField, q_out: FloatField):
     from __externals__ import i_end, i_start, j_end, j_start
 
     # Fills the same scalar value into three locations in q for each corner
-    # TODO(eddied): Alternating computations prevents improper fusion in the toolchain
-    with computation(PARALLEL), interval(...), horizontal(region[i_start, j_start]):
-        q_out = (q_in[0, 0, 0] + q_in[-1, 0, 0] + q_in[0, -1, 0]) * (1.0 / 3.0)
-    with computation(FORWARD), interval(...), horizontal(region[i_start - 1, j_start]):
-        q_out = q_in[1, 0, 0]
-    with computation(PARALLEL), interval(...), horizontal(region[i_start, j_start - 1]):
-        q_out = q_in[0, 1, 0]
-
-    with computation(FORWARD), interval(...), horizontal(region[i_end, j_start]):
-        q_out = (q_in[0, 0, 0] + q_in[1, 0, 0] + q_in[0, -1, 0]) * (1.0 / 3.0)
-    with computation(PARALLEL), interval(...), horizontal(region[i_end + 1, j_start]):
-        q_out = q_in[-1, 0, 0]
-    with computation(FORWARD), interval(...), horizontal(region[i_end, j_start - 1]):
-        q_out = q_in[0, 1, 0]
-
-    with computation(PARALLEL), interval(...), horizontal(region[i_end, j_end]):
-        q_out = (q_in[0, 0, 0] + q_in[1, 0, 0] + q_in[0, 1, 0]) * (1.0 / 3.0)
-    with computation(FORWARD), interval(...), horizontal(region[i_end + 1, j_end]):
-        q_out = q_in[-1, 0, 0]
-    with computation(PARALLEL), interval(...), horizontal(region[i_end, j_end + 1]):
-        q_out = q_in[0, -1, 0]
-
-    with computation(FORWARD), interval(...), horizontal(region[i_start, j_end]):
-        q_out = (q_in[0, 0, 0] + q_in[-1, 0, 0] + q_in[0, 1, 0]) * (1.0 / 3.0)
-    with computation(PARALLEL), interval(...), horizontal(region[i_start - 1, j_end]):
-        q_out = q_in[1, 0, 0]
-    with computation(FORWARD), interval(...), horizontal(region[i_start, j_end + 1]):
-        q_out = q_in[0, -1, 0]
+    with computation(PARALLEL), interval(...):
+        q_out = q_in
+
+        third = 1.0 / 3.0
```

Comment:
This silly trick was unfortunately necessary to avoid increasing the savepoint fortran-to-python error tolerance.